src/victory-util/add-events.js

```diff
@@ -0,0 +1,61 @@
+import { defaults, assign, isFunction, partialRight } from "lodash";
+import Events from "./events";
+
+export default (WrappedComponent) => {
+  return class addEvents extends WrappedComponent {
+    constructor() {
+      super();
+      this.state = {};
+      const getScopedEvents = Events.getScopedEvents.bind(this);
+      this.getEvents = partialRight(Events.getEvents.bind(this), getScopedEvents);
+      this.getEventState = Events.getEventState.bind(this);
+    }
+
+    componentWillMount() {
```

Comment:
With animations / transitions extracted into a the `VictoryTransitons` wrapper, I don't think victory components should really be stateful aside from events. 

I can add the errors though, or even `super.componentWillMount()` etc when those functions exist. if you think that's prudent.